terraform/helm/validator-lite/values.yaml

```diff
@@ -0,0 +1,83 @@
+chain:
+  name: devnet
+  era: 1
+  chain_id: 4
+
+imageTag: devnet
+nameOverride: aptos-validator
+
+validator:
+  name:
+  image:
+    repo: aptoslab/validator
+    tag:
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 4
+      memory: 6.5Gi
+    requests:
+      cpu: 4
+      memory: 6.5Gi
+  storage:
+    class:
+    size: 350Gi
+  rust_log: info
+  rust_log_remote: debug,hyper=off
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  config:
+    provide_genesis: true
+    sync_only: false
+    mempool_poll_count:
+    round_initial_timeout_ms:
+    default_prune_window: 10000000
+    state_store_prune_window: 1000000
+    max_version_to_prune_per_batch: 100
+
+vault:
+  image:
+    repo: vault
+    tag: 1.8.1@sha256:d35855569d87911f396bf951104ff6603ca1858ef2283719a935f8bcf9167e64
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 0.1
+      memory: 128Mi
+    requests:
+      cpu: 0.1
+      memory: 128Mi
+  server:
+    address:
+  serverIPRanges: []
+  prometheusTarget:
+  tls: {}
+  auth:
+    type: kubernetes
+    mount_path: auth/kubernetes-aptos
+    config:
+      role: aptos-<role>
+  namespace: aptos
+
+service:
+  external:
+    type: LoadBalancer
+  validator:
+    loadBalancerSourceRanges:
+  monitoring:
+    loadBalancerSourceRanges:
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+# Exposes the validator's REST API
+exposeValidatorRestApi: false
```

Comment:
should we default this to `true` to the txn emitter works out of the box?

Reply:
I'd rather not since we should be prioritizing for single node usage, instead of the multi-node testnet mode. the default value should be the one we recommend people to use for running a production validator node, so we'd need to revisit all the configs to make sure they're reasonable too.

Reply:
sgtm. I just also realized that we still have `.Values.load_test.enabled`, so we can just toggle that